app/presenters/licence_details_presenter.rb

```diff
@@ -69,12 +69,10 @@ def authorities
   end
 
   def authority
-    if authorities.size == 1
-      authorities_from_api_response.first
-    elsif authorities.size > 1 && local_authority_specific?
-      authorities_from_api_response.first
-    elsif authorities.size > 1 && authority_slug
+    if authorities.size > 1 && authority_slug
       authorities.detect { |a| a["slug"] == authority_slug }
+    elsif authorities.size == 1 || authorities.size > 1 && local_authority_specific?
```

Comment:
I think this would be a little clearer if there were two `elsif` clauses like before

Reply:
Thanks, would this work? :) 

```ruby
    elsif authorities.size == 1
      authorities_from_api_response.first
    elsif authorities.size > 1 && local_authority_specific?
      authorities_from_api_response.first
    end
```

Reply:
Yes, that would be good

Reply:
Cool, I made that change.